### PR TITLE
DOCS-1700: Update registry page

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/registry/_index.md [hunk 1]
```diff
@@ -28,10 +28,24 @@ menuindent: true
...
 The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:
 
-- [ML models](/ml/) to deploy with machine applications like [computer vision](/ml/vision/)
-- {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`
+{{< cards >}}
+{{% manualcard link="#ml-models" %}}
+
+#### ML models
+
+ML models to deploy with machine applications like computer vision.
```
Comment (npentrel) [inline docs/registry/_index.md:36]:
```suggestion
**ML models** to deploy with machine applications like computer vision.
```

File: docs/registry/_index.md [hunk 2]
```diff
@@ -28,10 +28,24 @@ menuindent: true
 
 The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:
 
-- [ML models](/ml/) to deploy with machine applications like [computer vision](/ml/vision/)
-- {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`
+{{< cards >}}
+{{% manualcard link="#ml-models" %}}
+
+**ML models**
+
```
Comment (npentrel) [inline docs/registry/_index.md:35]:
```suggestion
```

File: docs/registry/_index.md [hunk 3]
```diff
@@ -28,10 +28,24 @@ menuindent: true
...
+ML models to deploy with machine applications like computer vision.
+
+{{% /manualcard %}}
+{{% manualcard link="#modular-resources" %}}
+
+**Modular resources**
+
```
Comment (npentrel) [inline docs/registry/_index.md:42]:
```suggestion
```

File: docs/registry/_index.md [hunk 4]
```diff
@@ -28,10 +28,20 @@ menuindent: true
 
 The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:
 
-- [ML models](/ml/) to deploy with machine applications like [computer vision](/ml/vision/)
-- {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`
+{{< cards >}}
+{{% manualcard link="#ml-models" %}}
+
+ML models to deploy with machine applications like computer vision.
```
Comment (npentrel) [inline docs/registry/_index.md:34]:
```suggestion
<p><b>ML models</b> to deploy with machine applications like computer vision</p>
```

File: docs/registry/_index.md [hunk 5]
```diff
@@ -28,10 +28,20 @@ menuindent: true
...
+
+{{% /manualcard %}}
+{{% manualcard link="#modular-resources" %}}
+
+{{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`.
```
Comment (npentrel) [inline docs/registry/_index.md:39]:
```suggestion
<p>{{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to <code>viam-server</code></p>
```

File: docs/registry/_index.md [hunk 6]
```diff
@@ -28,10 +28,20 @@ menuindent: true
...
-- {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`
+{{< cards >}}
+{{% manualcard link="#ml-models" %}}
+
+**ML models** to deploy with machine applications like computer vision
+
+{{% /manualcard %}}
+{{% manualcard link="#modular-resources" %}}
+
+{{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`
```
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/registry/_index.md:39]: We can but we'd lose the glossary tooltip

[the remark this answers is not in the document]
Comment (nataliajacobowitz) [reply docs/registry/_index.md:39]: Then do you want to add a separate title to each card? Just helpful for me when scanning. Ill leave that decision to you as docs owner.

**Review by nataliajacobowitz — APPROVED** [review]
it looks good to me now thank you!

**Pull request merged**